src/reducers/PrescriptionReducer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import { ROUTES } from '../navigation/constants';
+import { UIDatabase } from '../database';
+import { createRecord } from '../database/utilities/index';
+
+const initialState = () => ({
+  currentTab: 0,
+  transaction: null,
+});
+
+const ACTIONS = {
+  SWITCH_TAB: 'PRESCRIPTION/SWITCH_TAB',
+  SELECT_ITEM: 'PRESCRIPTION/SELECT_ITEM',
+};
+
+export const switchTab = nextTab => ({
+  type: ACTIONS.SWITCH_TAB,
+  payload: { nextTab },
+});
+
+export const selectPrescriber = prescriberID => (dispatch, getState) => {
+  const { prescription } = getState();
+
```

Comment:
There's

A lot of

Line breaks here.

Reply:
0b8fdd389274ceaa8dffd858fa70f5de63172216